proposals/NNNN-android-notifications.md

```diff
@@ -0,0 +1,228 @@
+
+Some user complaints in Google Play Store for app integrating the latest version of the Android library in the past few months.
+
+1. **"...option to turn off SmartDeviceLink...App constantly popping up notifications...dont want this app doing anything in the background."**
+
+    ![Google Playstore review](../assets/proposals/NNNN-android-notifications/review_1.png)
+
+2. **"...quite good app but I've had to uninstall...I cann't even find any info about how to disable it..."**
+
+    ![Google Playstore review](../assets/proposals/NNNN-android-notifications/review_2.png)
+
+3. Translated: **"Privacy issue...It connects or analyzes my Polar M400 sports-watch without consent..."**
+
+    ![Google Playstore review](../assets/proposals/NNNN-android-notifications/review_3.png)
+
+**Feeback from App partners**
+
+Below comments on SDL `Waiting for connection` notifications received from App partners.
+
+1.  "...the constant notifications that kept appearing at the top of many users’ screens saying app is searching for connection...but given the amount of negative feedback this has generated from users via the Play store and Twitter, we have made the difficult decision to remove the functionality..."
+
+2. "...the notification shows for seconds and disappears, but this is quite a bad UX since all the users will be affected and the number of users that connect their devices to Bluetooth speakers/earphones is huge. We already received multiple bad reviews about this from our BETA testers. So, please consider fixing this asap..."
+
+Some Links from forum discussions in the past years. Similar to the above comments, the users in the below discussion forums complain about these SDL notifications.
+
+[Forum discussions 1](https://eu.community.samsung.com/t5/Other-Smartphones/S8-keep-getting-a-notification-about-smartdevicelink/td-p/542988)
+
+[Forum discussions 2](https://forums.androidcentral.com/ask-question/890465-how-get-rid-sdl.html)
+
+[Forum discussions 3](https://www.android-hilfe.de/forum/huawei-p20-pro.3297/smartdevicelink-meldung-deaktivieren.885217.html)
+
+
+This proposal describes a way in which app developers can control SDL behavior by using APIs to enable or disable it. It also proposes changes in the Android Router service to mitigate notifications issues.
+
+## Proposed solution
+
+Proposed solution is described below.
+
+### Provide API for app developers to enable/disable SDL in Android proxy
+
+The Android proxy should provide an API to enable-disable SDL. The router service should be enabled by default. This would help many app developers to provide a setting in their app so that users can disable it if they don't have access to an SDL enabled system. This API will not influence the behavior of other SDL apps available on the users' device.
```

Comment:
```suggestion
The Android proxy should provide an API to enable/disable SDL. The router service should be enabled by default. This would help many app developers to provide a setting in their app so that users can disable it if they don't have access to an SDL enabled system. This API will not influence the behavior of other SDL apps available on the users' devices.
```